src/app/core/models/session.model.ts

```diff
@@ -11,31 +11,52 @@ export interface SessionData {
 }
 
 export const SESSION_DATA_UNAVAILABLE = 'Session Data Unavailable';
-export const SESSION_DATA_INCORRECTLY_DEFINED = 'Session Data is Incorrectly Defined';
+export const SESSION_DATA_MISSING_CRUCIAL_INFO = 'Session Data is missing crucial components';
```

Comment:
> Hmm that could be confusing because the first one is missing the openPhases key and the second has no open phases. The crucial info here refers to the first case. Perhaps `Session Data does not define openPhases`

Let's go with `Session data does not define an 'openPhases' key`.